environment_setup/setup_software.sh

```diff
@@ -102,6 +102,11 @@ if ! sudo apt-get install "${host_software_packages[@]}" -y ; then
     exit 1
 fi
 
+if [[ $(lsb_release -rs) == "18.04" ]]; then
+    sudo rm /usr/bin/python3 # remove symlink
+    sudo ln -s python3.8 /usr/bin/python3 # update symlink to python3.8
+fi
+
```

Comment:
why are we messing with `/usr/bin/python3` when we have /opt/tbotspython?

Reply:
its because the python3 virtual env doesn't have the headers needed by pybind11 and I thought I had exhausted all options. This also is changing the symlink to be a newer version

that being said, I found a better way through the `.bazelrc` so removed here